lacci/lib/shoes/builtins.rb

```diff
@@ -2,59 +2,61 @@
 
 # Shoes has a number of built-in methods that are intended to be available everywhere,
 # in every Shoes and non-Shoes class, for every Shoes app.
-class Shoes::Builtins
+module Shoes::Builtins
   # Register the given font with Shoes so that text that wants it can use it.
   # Also add it to the FONTS constant.
   #
   # @param font_file_or_url [String] the filename or URL for the font
   # @return [void]
-  def font(font_file_or_url)
-    shoes_builtin("font", font_file_or_url)
+  class Shoes::Builtins
```

Comment:
i think u r supposed to change class back to module not adding module on top of class

Reply:
yeah ,I have changed it